tests/RuleSet/RuleSetTest.php

```diff
@@ -473,11 +468,11 @@ private function expendSet(array $setDefinitions, array $resolvedSets, string $s
         return $resolvedSets[$setName];
     }
 
-    private static function assertSameRules(array $expected, array $actual, string $message = ''): void
```

Comment:
why dropping the message?

Reply:
because it was never used, so less code to maintain